### ISSUE TITLE
Validation Set and ChamferL1

### ISSUE THREAD

**Issue by andrearosasco** [yuxumin/PoinTr#62]
Hi, I was wondering how you handled validation.

In some different configuration files I see:
```
dataset : {
  train : { _base_: cfgs/dataset_configs/PCN.yaml, 
            others: {subset: 'train'}},
  val : { _base_: cfgs/dataset_configs/PCN.yaml, 
            others: {subset: 'test'}},
  test : { _base_: cfgs/dataset_configs/PCN.yaml, 
            others: {subset: 'test'}}}
```

does it mean you are using the test set as a validation set? If not, how big is the validation set you used?

Also, `ChamferDistanceL1()` returns the average of the two chamfer distance components:
`torch.mean(dist1) + torch.mean(dist2))/2`
Is that correct? Are the results reported for PCN computed like that?

Thanks in advance

**Comment by andrearosasco** [comment]
Alright, thanks! I want to make some comparisons and wondered how I should compute the metric 👍

**Comment by yuxumin** [comment]
Hi, 
To fairly compare with the previous work, we train the model on training set and validate it on test set as most previous work does. And it is important to check the verison of Chamfer-Distance is used (L1 or L2). As you have mentioned. L1-version CD needs to be multiplied with 0.5, while L2-version is not required to be scaled.

**Comment by andrearosasco** [comment]
Alright, all clear now, thank you!
